third_party/egeria/usage/read_asset.sh

```diff
@@ -8,22 +8,23 @@ echo "Read asset info"
 
 echo "ASSET_ID: $GUID"
 
-USER=calliequartile
+USER=garygeeke
```

Comment:
Same comment about hard-coded username. Should not be in the git repo.

Reply:
This file is not used during deployment. It is mainly for initialisation purposes on local machine. So it is not possible to read this value from deployment based env variable in my view. Please suggest if there are any other ways to initialise this variable. Thanks.